rides/templates/index.html

```diff
@@ -187,5 +187,9 @@ <h5 class="modal-title">Delete Event</h5>
       {% endfor %}
       {% if current_user.is_authenticated %}<a class="btn btn-primary btn-lg btn-block" align="center" href="/eventform" id="new">Create
       Event</a>{% endif %}
+      {% if current_user.acc_type == "CSH" and current_user.slack == "" %}
+      <h4 align="center">Please ensure your Slack is synced at <a href = "https://eac.csh.rit.edu">https://eac.csh.rit.edu</a>!</h4>
+      <h4 align="center">Re-Login once you have synced to apply changes!</h4>
+      {% endif %}
```

Comment:
Curious, what does this end up looking like? Could I get a screenshot?

Reply:
How about something more akin to "Please ensure your Slack is linked on [EAC](https://eac.csh.rit.edu)!"? It's kinda weird to have the complete link there...

Reply:
<img width="1027" alt="image" src="https://user-images.githubusercontent.com/40615740/135873248-e0c0f016-1a19-4397-918b-a65cd3fe0ce7.png">

Reply:
Hmm. I think this would be better as a banner. I think packet and impeach both use some bootstrap styles for doing that if you want examples.